LoopKit/Persistence/PersistenceController.swift

```diff
@@ -177,7 +177,7 @@ public final class PersistenceController {
                 }
             }
 
-            let storeURL = directoryURL.appendingPathComponent("Model.sqlite")
+            let storeURL = directoryURL.appendingPathComponent("Model4.sqlite")
```

Comment:
I created multiple models as I went through the development process; I'm guessing it would be appropriate to get rid of models 2 & 3, and have the jump be from model.sqlite -> model4.sqlite?